cirq/ops/common_channels.py

```diff
@@ -184,6 +195,700 @@ def depolarize(p: float) -> DepolarizingChannel:
             the Pauli gates is applied independently with probability p / 3.
 
     Raises:
-        ValueError if p is not a valid probability.
+        ValueError: if p is not a valid probability.
     """
     return DepolarizingChannel(p)
+
+
+class GeneralizedAmplitudeDampingChannel(raw_types.Gate):
+    """Dampen qubit amplitudes through non ideal dissipation.
+
+    This channel models the effect of energy dissipation to the
+    surrounding environment at some particular rate, with some
+    given probability at a non zero temperature.
+    """
+
+    def __init__(self, p, gamma) -> None:
+        """The generalized amplitude damping channel.
+
+        Construct a channel to model the energy dissipation to the
+        environment at rate gamma, occuring with probability p
```

Comment:
nit: period at end of comments (here and in args)

Reply:
fixed, added periods everywhere I could.